packages/p2p/src/pages/advertiser-page/advertiser-page.jsx

```diff
@@ -126,14 +128,14 @@ const AdvertiserPage = () => {
     );
 
     React.useEffect(() => {
-        if (is_advertiser && !is_barred) {
+        if (is_advertiser && !is_barred && is_my_advert !== null && !is_my_advert) {
```

Comment:
I'm a bit confused about why are we explicitly checking for `not null` & `!is_my_advert` 😅

Reply:
if i dont check for null, this will still show the modal since `is_my_advert` can be null and if i just do the check for `!is_my_advert` when it is null, it will return true 😓